### PR TITLE
List stacks: pagination not implemented

### PR DESCRIPTION
**Pull request by mweibel**

### CODE REVIEW

File: source/_references/stacks.md [hunk 1]
```diff
@@ -447,25 +447,31 @@ if (true === $deleted) {
...
 ```language-bash
-curl -X GET 'https://api.rokka.io/stacks/testorganization'
+curl -X GET 'https://api.rokka.io/stacks/testorganization' -H 'Api-Key: key'
 ```
 
-```language-php
-use Rokka\Client\Core\Stack;
-
-$client = \Rokka\Client\Factory::getImageClient('testorganization', 'apiKey', 'apiSecret');
-
```
Comment (chregu) [inline source/_references/stacks.md]: why did you remove the PHP example? We don't have any other place, where the PHP API is described in full (may be something we should do some day, of course)

**Comment by dbu** [comment]
do we try to drop all php examples? or a specific reason to drop that here? i think having examples in curl and with all libs we have would be desirable.

**Comment by mweibel** [comment]
Removed it just because ;) WIll re-add it. 
We need to think about a solution to show all clients. Can we do that with sculpin even?

**Comment by dbu** [comment]
> We need to think about a solution to show all clients. Can we do that with sculpin even?

i don't think sculpin has that built-in. i love sphinx for writing docs. in symfony they built a sphinx plugin that gives you tabs with all possible options, e.g. this yaml / xml tabs: http://symfony.com/doc/master/reference/configuration/doctrine.html
